perf/lua/box_select.lua

```diff
@@ -1,27 +1,34 @@
 --
 -- The test measures run time of various space select operations.
 --
--- Output format:
--- <test-case> <run-time-nanoseconds>
+-- Output format (console):
+-- <test-case> <iterations-per-second>
 --
 -- Options:
--- --pattern <string>  run only tests matching the pattern; it's possible
---                     to specify more than one pattern separated by '|',
---                     for example, 'get|select'
--- --read_view         use a read view
+-- --pattern <string>       run only tests matching the pattern;
+--                          it's possible to specify more than one
+--                          pattern separated by '|', for example,
+--                          'get|select'
+-- --read_view              use a read view
 --
+-- --output <string>        file to output
+-- --output_format <string> console (default)|json
 
 local clock = require('clock')
 local fiber = require('fiber')
 
 local params = require('internal.argparse').parse(arg, {
+    {'output', 'string'},
+    {'output_format', 'string'},
     {'pattern', 'string'},
     {'read_view', 'boolean'},
 })
```

Comment:
Maybe wrap `argparse.parse()` into some `benchmark.parse_args()` function to add more options consistently to all the benchmarks?

Reply:
Added a wrapper, so the usage is the following:
```lua
local benchmark = require('benchmark')

local opts = benchmark.argparse(arg, {--[[custom options]]})
local bench = benchmark.new(opts)
```